### ISSUE TITLE
How to excute "axaml" file in avalonia ilspy?

### ISSUE THREAD

**Issue by RickyYChan** [icsharpcode/AvaloniaILSpy#70]
I lost the source code of one of my small project, so i decided to decompiled the dll.
![shot](https://user-images.githubusercontent.com/77863853/117665844-d6831f80-b1d5-11eb-91db-f51b676edf97.png)
So how to excute or see the xaml code in the project? (mainwindowview.axaml)
:)

**Comment by christophwille** [comment]
ILSpy has a decompiler for BAML - maybe something similar would need to be built for axaml (given how few Avalonia apps are out there this is really a niche problem).

**Comment by RickyYChan** [comment]
> ILSpy has a decompiler for BAML - maybe something similar would need to be built for axaml (given how few Avalonia apps are out there this is really a niche problem).

Thank you! :D When searching the problem on bing, everyone said that you should have a look on baml file, but indeed, there's no baml file of axaml in avalonia. AvaloniaUI is rarely used by people (compared to WPF), so it is difficult to find a answer to this problem.

**Comment by RickyYChan** [comment]
Obviously, this file contains the full message of our axaml file. So if we can transform this into a axaml file, then we dont need the baml file like what we did in wpf.
![quick](https://user-images.githubusercontent.com/77863853/117676434-1f3fd600-b1e0-11eb-97a5-8c81c64c91c0.png)

**Comment by unofficialdev** [comment]
I'm looking forward to this too
